src/features/list-view/ListView.js

```diff
@@ -22,41 +22,42 @@ type Props = {
     getGridHeader: (columnIndex: number) => any,
     getGridHeaderSort?: (columnIndex: number) => typeof SORT_ORDER_ASCENDING | typeof SORT_ORDER_DESCENDING | null,
     height: number,
+    isGridHeaderSortable?: (columnIndex: number) => boolean,
     onSortChange?: (columnIndex: number) => void,
     rowCount: number,
     width: number,
 };
 
+const CSS_PREFIX = 'bdl-MetadataViews-ListView';
+
 class ListView extends React.PureComponent<Props> {
     cellRenderer = ({ columnIndex, key, rowIndex, style }: CellRendererArgs) => {
-        const { getGridCell, getGridHeader, getGridHeaderSort = noop, onSortChange = noop } = this.props;
+        const { getGridCell, getGridHeader, isGridHeaderSortable = noop, onSortChange = noop } = this.props;
 
         if (rowIndex === 0) {
             const displayName = getGridHeader(columnIndex);
-            const sortDirection = getGridHeaderSort(columnIndex);
-            const isSortAsc = sortDirection === SORT_ORDER_ASCENDING;
-            const className = classNames({
-                'bdl-ListView-isSortAsc': isSortAsc,
-            });
+            const isSortable = isGridHeaderSortable(columnIndex);
 
             return (
                 <button
-                    className="bdl-ListView-columnHeader"
+                    className={classNames(`${CSS_PREFIX}-columnHeader`, {
```

Comment:
This seems like a pretty heavy and somewhat low value abstraction for a cell that could be rendered many times. I believe this generates/calls functions under the hood after transpilation.

Reply:
Not sure if I agree on the heaviness.

Reply:
Such multiline class names should also be taken out from within JSX so that while debugging we can put proper break points